server/graphql/utils/requestType.ts

```diff
@@ -1,27 +1,66 @@
-import { ServerResponseInterface } from '../serverResponse'
+import { Document, Types } from 'mongoose'
+import { softDeleteRequestHelper } from './request'
 import { updateRequestGroupHelper } from './requestGroup'
 
 import { UserInputError } from 'apollo-server-errors'
 
-const updateRequestTypeHelper = (requestType, dataSources): Promise<ServerResponseInterface> => {
+import mongoose from 'mongoose'
+
+const updateRequestTypeHelper = async (requestType, dataSources): Promise<Document> => {
   if(!requestType.id) {
     throw new UserInputError('Missing argument value', { argumentName: 'id' })
   }
-  const requestGroupId = dataSources.requestTypes.getById(requestType.id.toString()).requestGroup
-  return dataSources.requestTypes.update(requestType)
-    .then(res => {
-      updateRequestGroupHelper({"id": requestGroupId}, dataSources)
-      return res
-    })
-    .catch(error => {
-      return error
-    })
+  const session = await mongoose.startSession()
+  try {
+    session.startTransaction()
+    const currentRequestType = dataSources.requestTypes.getById(requestType.id.toString())
```

Comment:
Similar to another comment. Ideally, we do not need to store old versions within a transaction